### PR TITLE
Add support for dashboard variables in transformations to cloud what's new

### PR DESCRIPTION
**Pull request by imatwawana**

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 1]
```diff
@@ -18,6 +18,17 @@ weight: -37
...
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## Support for dashboard variables in transformations
+
+<!-- Oscar Kilhed, Victor Marin -->
+<!-- already in on-prem -->
+
+_Experimental in Grafana Cloud_
+
+Previously, the only transformation that supported [dashboard variables](https://grafana.com/docs/grafana/<GRAFANA VERSION>/dashboards/variables/) was the **Add field from calculation** transformation. We've now extended the support for variables to the **Filter by value**, **Create heatmap**, **Histogram**, **Sort by**, **Limit**, **Filter by name**, and **Join by field** transformations. We've also made it easier to find the correct dashboard variable by displaying available variables in the fields that support them, either in the drop-down or as a suggestion when you type **$** or press Ctrl + Space:
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:28]:
```suggestion
Previously, the only transformation that supported [dashboard variables](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/dashboards/variables/) was the **Add field from calculation** transformation. We've now extended the support for variables to the **Filter by value**, **Create heatmap**, **Histogram**, **Sort by**, **Limit**, **Filter by name**, and **Join by field** transformations. We've also made it easier to find the correct dashboard variable by displaying available variables in the fields that support them, either in the drop-down or as a suggestion when you type **$** or press Ctrl + Space:
```

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 2]
```diff
@@ -18,6 +18,17 @@ weight: -37
...
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## Support for dashboard variables in transformations
+
+<!-- Oscar Kilhed, Victor Marin -->
+<!-- already in on-prem -->
+
+_Experimental in Grafana Cloud_
+
+Previously, the only transformation that supported [dashboard variables](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/dashboards/variables/) was the **Add field from calculation** transformation. We've now extended the support for variables to the **Filter by value**, **Create heatmap**, **Histogram**, **Sort by**, **Limit**, **Filter by name**, and **Join by field** transformations. We've also made it easier to find the correct dashboard variable by displaying available variables in the fields that support them, either in the drop-down or as a suggestion when you type **$** or press Ctrl + Space:
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:28]:
```suggestion
Previously, the only transformation that supported [dashboard variables](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/dashboards/variables/) was the **Add field from calculation** transformation. We've now extended the support for variables to the **Filter by value**, **Create heatmap**, **Histogram**, **Sort by**, **Limit**, **Filter by name**, and **Join by field** transformations. 

We've also made it easier to find the correct dashboard variable by displaying available variables in the fields that support them, either in the drop-down or as a suggestion when you type **$** or press Ctrl + Space:
```

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 3]
```diff
@@ -18,6 +18,19 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## Support for dashboard variables in transformations
+
+<!-- Oscar Kilhed, Victor Marin -->
+<!-- already in on-prem -->
+
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:25]:
```suggestion

October 24, 2023

```

**Review by oscarkilhed — APPROVED** [review]
:tada: Thank you!

**Pull request merged**